src/components/TagCloud/TagCloud.js

```diff
@@ -0,0 +1,26 @@
+import React, { PropTypes } from 'react'
+import {Tag} from '../Tag'
+
+import {styles} from './styles'
+
+const TagCloud = ({names, onClickAction}) => (
+  <div>
+    {
+      names.map((name, i) => (
+       <div {...styles.item}>
+         {onClickAction
+           ? <Tag key={i} name={name} onClickAction={(name) => onClickAction(name)} />
```

Comment:
@heinburger I know this is closed but
What happens if we display a tag name, but  we remove it based on its resource_uri? does `onClickAction(name)` work?